### PR TITLE
Add salary req in ReadMe

### PR DESCRIPTION
**Pull request by MLGabe**

### CODE REVIEW

**Review by kspaans — APPROVED** [review]

**Review by theetrain — COMMENTED** [review]
Do we want it in the subject? Plus companies tend to offer ranges; so including it in the body should be enough, I think.

**Comment by MLGabe** [comment]
> Do we want it in the subject? Plus companies tend to offer ranges; so including it in the body should be enough, I think.

Hm.. 
I like it being on the body as it's easier to see from Slack, but am not married to the idea.
Regarding ranges I think it's fine as well, the issue is that sometimes the range is HUGE.

**Comment by Soviut** [comment]
One thing we should make sure is they include the currency for the salary. It can be pretty confusing when USD or CAD isn't mentioned.

File: README.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ Please note: we strongly believe that **great companies always start with a frie
 Every post title **MUST** follow the format below:
 
 ```
-[Company Name] - [Job Title] - [Location]
+[Company Name] - [Job Title] - [Location] - [Salary]
```
Comment (Soviut) [inline README.md:18]:
```suggestion
[Company Name] - [Job Title] - [Location] - [Salary] [Currency]/[Frequency]
```

Comment (Soviut) [reply README.md:18]: As per my comment in the main thread, I think showing they should add currency is a good idea.

**Comment by MLGabe** [comment]
> > Do we want it in the subject? Plus companies tend to offer ranges; so including it in the body should be enough, I think.
> 
> Hm.. I like it being on the body as it's easier to see from Slack, but am not married to the idea.

Here I meant that I like it being in the Subject :D

**Pull request merged**